src/core/DisplayGroup.h

```diff
@@ -109,6 +131,12 @@ public slots:
      */
     void setBackgroundContent( ContentPtr content );
 
+    /**
+     * Set the the value of the showWindowTitles option.
+     * @param set The setter for showWindowTitles option.
```

Comment:
yes, I meant the parameter. The function sets the value, and the value is on/off or enable/disable or something like that.  On the other hand, in Options.h they are also called 'set', so for the sake of consistency you can leave it.